medusa/storage/s3_rgw.py

```diff
@@ -33,7 +33,7 @@ def connect_storage(self):
         driver = S3RGWStorageDriver(
             host=self.config.host,
             port=self.config.port,
-            region="default",
+            region="default" if credentials['region'].lower() in ('0', 'false') else credentials['region'],
```

Comment:
This makes sense to me to make the region configurable specially for medusa (no need to change the default).

Yet, I would change that code a bit, what about this instead:
```
region="default" if not credentials['region'] else credentials['region'].lower()
```

Would you test it like this (check if we have a value, if we have it, lowercase-it)?

Reply:
I'm not sure what you're aiming at with: `if credentials['region'].lower() in ('0', 'false')`, would you explain (if you think your version is better)?

Reply:
Hi,
Thanks for your comment.I think I found a better solution, "region" must be declared on [storage] section and not on ceph-credentails. it makes more sense to me i think